docs/conf.py

```diff
@@ -19,6 +19,7 @@
 
 project = 'Gro API Client'
 html_logo = '_images/logo.jpg'
+# html_logo = '_images/Gro-logo-full.svg'
```

Comment:
temporally put red logo back , to avoid undesired document site change before we're ready